lightning/src/routing/scoring.rs

```diff
@@ -3516,8 +3458,6 @@ mod tests {
 			};
 
 			assert_eq!(scorer.channel_penalty_msat(&candidate, usage, &params), 2050);
-			usage.inflight_htlc_msat = 0;
-			assert_eq!(scorer.channel_penalty_msat(&candidate, usage, &params), 866);
```

Comment:
Was just fixing a build warning and noticed this. Why did this check need to be removed? Deleted in 35b49645c4eb0d61c162a502723cd4e44ef0ca1e.